### DISCUSSION TITLE
How i style navigation arrows in core?

### DISCUSSION THREAD

**Discussion by santosh-marar** [nolimits4web/swiper#6229] [Q&A]
.swiper-button-next,
.swiper-button-prev {
  color: white;
}
 is not working please tell me how i change color of .swiper-button-next,
.swiper-button-prev and also .swiper-pagination

**Comment by Arhell** [comment]
Hi
in the core it is better not to change, reuse styles
example in custom.css 
```
.swiper-button-next,
.swiper-button-prev {
color: white !important;
}
```

or 

```
.someClass .swiper-button-next,
.someClass .swiper-button-prev {
color: white ;
}
```
